### PR TITLE
feat(mobile-rules-guidance): add rule link

### PR DESCRIPTION
**Pull request by haonliu**
#### Description of changes

Adding rule link for each mobile rules.
screenshot:
image

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/electron/platform/android/rule-information-provider.ts [hunk 1]
```diff
@@ -14,20 +14,23 @@ export class RuleInformationProvider {
...
                 'TouchSizeWcag',
+                'https://accessibilityinsights.io/info-examples/android/touch-size-wcag/',
                 'Touch inputs must have a sufficient target size.',
                 [link.WCAG_1_3_1, link.WCAG_3_3_2],
                 this.getTouchSizeUnifiedFormattableResolution,
                 this.includeAllResults,
             ),
             ActiveViewName: new RuleInformation(
                 'ActiveViewName',
+                'https://accessibilityinsights.io/info-examples/android/touch-size-wcag/',
```
Comment (karanbirsingh) [inline src/electron/platform/android/rule-information-provider.ts:33]: should be https://accessibilityinsights.io/info-examples/android/active-view-name

File: src/electron/platform/android/rule-information-provider.ts [hunk 2]
```diff
@@ -14,20 +14,23 @@ export class RuleInformationProvider {
         this.supportedRules = {
             ColorContrast: new RuleInformation(
                 'ColorContrast',
+                'https://accessibilityinsights.io/info-examples/android/color-contrast/',
```
Comment (karanbirsingh) [inline src/electron/platform/android/rule-information-provider.ts:17]: is it worth sharing the base URL for all of these in a common variable?

File: src/electron/platform/android/rule-information-provider.ts (hunk 1 above)
Comment (RobGallo) [inline src/electron/platform/android/rule-information-provider.ts:33]: Shouldn't this be a link to ActiveViewName?

File: src/tests/unit/tests/electron/platform/android/scan-results-helpers.ts [hunk 3]
```diff
@@ -112,12 +112,14 @@ export function buildViewElement(
 
 export function buildRuleInformation(
     ruleId: string,
+    ruleLink = 'rule-link',
```
Comment (RobGallo) [inline src/tests/unit/tests/electron/platform/android/scan-results-helpers.ts:115]: Just curious, what is the difference in our case between the rule link and the guidance links? I thought this was guidance links. 😕

Comment (haonliu) [reply src/tests/unit/tests/electron/platform/android/scan-results-helpers.ts:115]: My understanding is rule link is the place where the rule is defined and explained, and guidance links provide some extra infomation.

**Pull request merged**